example/react/package.json

```diff
@@ -12,6 +12,9 @@
     "react-scripts": "5.0.1",
     "tailwindcss": "^1.8.10"
   },
+  "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11"
+  },
```

Comment:
What is this used for?

Reply:
it's just a dependency that needs to be added to remove a warning in the react example